cfg/cs2fixes/cs2fixes.cfg

```diff
@@ -39,4 +39,6 @@ zr_infect_spawn_type			1		//Type of Mother Zombies Spawn [0 = MZ spawn where the
 zr_infect_spawn_time_min		15		//Minimum time in which Mother Zombies should be picked, after round start
 zr_infect_spawn_time_max		15		//Maximum time in which Mother Zombies should be picked, after round start
 zr_infect_spawn_mz_ratio		7		//Ratio of all Players to Mother Zombies to be spawned at round start
-zr_infect_spawn_mz_min_count	2		//Minimum amount of Mother Zombies to be spawned at round start
\ No newline at end of file
+zr_infect_spawn_mz_min_count	2		//Minimum amount of Mother Zombies to be spawned at round start
+zr_respawn_delay				5.0		//Time before a zombie is respawned
+zr_default_winner_team			1		//Which team wins when time ran out, use 1 for draw
```

Comment:
I would do the description like `zr_infect_spawn_type` and show all the values. Most people won't know team numbers for T/CT.